docs/make_docs.py

```diff
@@ -301,6 +303,12 @@ def run(self):
         """
         Call Sphinx command with hard-coded "html" target
         """
+        # Copy docs files from Open3D-ML repo
+        OPEN3D_ML_ROOT = os.environ.get("OPEN3D_ML_ROOT", "../../Open3D-ML")
```

Comment:
Since we have `self.current_file_dir`, shall we make `../../Open3D-ML` relative to current file_dir?